sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py

```diff
@@ -0,0 +1,1393 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# mypy: disable-error-code="attr-defined"
+
+from typing import Any, List, Union, overload, Optional, cast, Tuple, TypeVar, Dict
+from enum import Enum
+import json
+import datetime
+
+from azure.core import CaseInsensitiveEnumMeta
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.paging import ItemPaged
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+from azure.core.pipeline.policies import HttpLoggingPolicy
+from azure.core.exceptions import HttpResponseError, ODataV4Format
+from azure.core.pipeline import PipelineResponse
+from azure.core.rest import (
+    HttpResponse,
+    AsyncHttpResponse,
+    HttpRequest,
+)
+from azure.core.polling import LROPoller
+from azure.core.polling.base_polling import (
+    LROBasePolling,
+    OperationResourcePolling,
+    _is_empty,
+    _as_json,
+    BadResponse,
+    OperationFailed,
+    _raise_if_bad_http_status_and_method,
+)
+
+from .models import (
+    TranslationStatus,
+    DocumentStatus,
```

Comment:
Why do we need this import statement and the one on L46-49?

Reply:
The last few PRs took care of this